analysis/parameter_tuning.py

```diff
@@ -26,6 +28,9 @@
 from enum import Enum
 import numpy as np
 
+from pipeline_dp.private_contribution_bounds import \
+    _generate_possible_contribution_bounds
```

Comment:
Nit1: please remove leading '_' in `_generate_possible_contribution_bounds` (it's not private anymore)
Nit2: please keep import in 1 line (it's ok for import to be longer than 80 chars)